test/run-test.sh

```diff
@@ -22,8 +22,7 @@ prefix="$(md5sum "${work_dir}/initramfs/check-net.sh" | cut -d' ' -f1)"
     -display none \
     -no-user-config \
     -nodefaults \
-    --enable-kvm \
-    -cpu host \
+    -cpu qemu64 \
```

Comment:
Hi, why it is better? Is this because you don't want to use sudo or any special privileges?

Reply:
Hi. When KVM is disabled this VM can be stated in a CI environment. So, we can add a test for CI for the AutoHCK installer.